File: src/main/java/ru/citeck/ecos/codeinsight/forms/components/Container.java
package ru.citeck.ecos.codeinsight.forms.components;

import java.util.ArrayList;
import java.util.List;

public abstract class Container extends Component {

    private List<Component> components = new ArrayList<>();

    public List<Component> getComponents() {
        return components;
    }

    public void setComponents(List<Component> components) {
        this.components = components;
    }


}

File: src/main/kotlin/ru/citeck/idea/artifacts/action/codeinsight/forms/components/Container.kt
package ru.citeck.idea.artifacts.action.codeinsight.forms.components

abstract class Container : ru.citeck.idea.artifacts.action.codeinsight.forms.components.Component() {
    var components: List<ru.citeck.idea.artifacts.action.codeinsight.forms.components.Component> = ArrayList()
}
